galley/pkg/server/server.go

```diff
@@ -210,16 +194,40 @@ func (s *Server) Close() error {
 		_ = s.listener.Close()
 	}
 
-	if s.livenessProbe != nil {
-		_ = s.livenessProbe.Close()
-	}
+	// final attempt to purge buffered logs
+	_ = log.Sync()
+
+	return nil
+}
+
+//RunServer start Galley Server mode
+func RunServer(sa *Args, printf, fatalf shared.FormatFn, livenessProbeController,
+	readinessProbeController probe.Controller) error {
```

Comment:
Consider removing the `error` return value here. `RunServer` is called as a goroutine, the return value is never checked, and fatal errors are reported by calling `fatalf` instead.

Reply:
done